tests/test_integrated_channels/test_sap_success_factors/test_client.py

```diff
@@ -340,11 +340,14 @@ def test_client_uses_prevent_learner_submit_flag(self):
         )
 
         expected_response = 200, json.dumps(expected_response_body)
+        transformed_payload = self.completion_payload
+        transformed_payload['courseCompleted'] = True
```

Comment:
Any concerns about `True` versus `true`?

Reply:
Discussed out-of-band -- we are sending `true` in the payload versus `True` -- good to go!

Reply:
Yeah just to clarify this exact test case:
The actual code logic takes a payload formatted as `completedCourse: "true"` and loads it which gives us `completedCourse: True` and then dumps it back out to get `completedCourse: true`

The 2 lines above here and the one below this comment mimic that logic to form the payload we should be sending to SAP (and in this test that we send to the `_call_post_with_session` method)

Reply:
It(the test) just has slightly different logic from the actual code because completion_payload here is a dictionary whereas in the code we would receive a serialized version (aka a string)